packages/default/scss/tooltip/_layout.scss

```diff
@@ -24,20 +30,33 @@ $tooltip-callout-size: 12px !default;
     }
 
     .k-tooltip-closable {
-        padding: $tooltip-closable-padding;
+        padding: $tooltip-closable-padding-y $tooltip-closable-padding-x;
+        line-height: normal;
+
+        .k-tooltip-content {
+            padding-right: $tooltip-closable-content-padding;
+
+            .k-rtl &,
+            [dir-rtl] & {
+                padding-left: $tooltip-closable-content-padding;
+                padding-right: 0;
+            }
+        }
     }
 
     .k-tooltip-button {
-        margin-left: $tooltip-padding-x;
+        position: absolute;
```

Comment:
The flex positioning is overwritten by the inbuilt javascript by "display: block". To prevent using "!important" I prefer to use the absolute position approach. I have removed the redundant lines.